pkg/cluster/k8sres.go

```diff
@@ -1183,7 +1196,7 @@ func (c *Cluster) generateStatefulSet(spec *acidv1.PostgresSpec) (*appsv1.Statef
 		effectiveRunAsUser,
 		effectiveRunAsGroup,
 		effectiveFSGroup,
-		nodeAffinity(c.OpConfig.NodeReadinessLabel),
+		nodeAffinity(c.OpConfig.NodeReadinessLabel, &spec.NodeAffinity),
```

Comment:
as mentioned above, this isn't done for the pooler pod template. Another question would be, if we should apply `c.OpConfig.NodeReadinessLabel`, too?

Reply:
Is that a question for me or for other members of postgres-operator team?